packages/kotti-ui/source/kotti-style/_layout.scss

```diff
@@ -125,6 +125,12 @@
 	margin-right: auto;
 }
 
+.col-with-row-gap {
```

Comment:
I don’t think we should be exposing more classes from Kotti.

Reply:
change amended

Reply:
to the wrong file, another change about to be pushed